### ISSUE TITLE
labcli errors in SSH, SCP environment variables?

### ISSUE THREAD

**Issue by bdlink** [cgruver/kamarotos#1]
In SSH devinition, PubkeyAcceptedAlgorithms does not seem to exist, should be PublickeyAcceptedKeyTypes.

in SCP definition, option -O does not seem to exist, removing it works.

Both of these are wrt archive branch

Bruce

**Comment by cgruver** [comment]
Yeah, the behavior has probably changed.  That branch is likely getting stale.
